docs/api_information/status_codes.md

```diff
@@ -22,6 +22,12 @@ Failed | FPRA04 | ProcessAuthorisation | Declined because the customer limit has
 Failed | FPRA05 | ProcessAuthorisation | Declined due to negative payment history for the customer
 Failed | FPRA06 | ProcessAuthorisation | Declined because the credit-card used for the deposit is expired
 Failed | FPRA07 | ProcessAuthorisation | Declined because supplied posTransactionRef has already been processed
+Failed | FPRA08 | ProcessAuthorisation | Risk Declined - The instalment amount was below the minimum threshold
```

Comment:
would prefer message in the same format as the other messages (i.e. "Declined because the instalment amount was below the minimum")